geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/FileResultModel.java

```diff
@@ -33,38 +35,46 @@
 public class FileResultModel {
   public static final int FILE_TYPE_BINARY = 0;
   public static final int FILE_TYPE_TEXT = 1;
+  public static final int FILE_TYPE_FILE = 2;
```

Comment:
should use enum rather than int constants, and add comment clarifying when each file type is applicable

Reply:
Yes, Enum would be better, but this PR is not refactoring to this level yet.